Amqp.V1/Akka.Streams.Amqp.V1/AmqpSinkStage.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Akka.Streams.Stage;
+using Amqp;
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace Akka.Streams.Amqp.V1
+{
+    public sealed class AmqpSinkStage<T> : GraphStageWithMaterializedValue<SinkShape<T>, Task>
+    {
+        public readonly Inlet<T> In = new Inlet<T>("AmqpSink.in");
+        public override SinkShape<T> Shape => new SinkShape<T>(In);
```

Comment:
It's most likely that the getter is called multiple times during materialization, i.e. wenn calling `.Run`, so  this should be set once in the constructor.